meson.build

```diff
@@ -41,7 +41,18 @@ if git.found()
 endif
 add_project_arguments('-DPROJECT_VERSION=@0@'.format(version), language: 'c')
 
-libdrm_inc = dependency('libdrm').partial_dependency(compile_args: true)
```

Comment:
Let's keep it simple. If the build option is set, use that include path regardless. If it's not set, do the partial dependency stuff like it's done here in this line.